src/vcpkg/commands.install.cpp

```diff
@@ -114,9 +114,22 @@ namespace vcpkg
                     if (fs.exists(target, IgnoreErrors{}))
                     {
                         msg::println_warning(msgOverwritingFile, msg::path = target);
+                        fs.remove_all(target, IgnoreErrors{});
+                    }
+                    static bool use_hard_link = true;
```

Comment:
I don't think a global is a reasonable answer here. This can be a local created just before the for loop?

Reply:
Yes. My thought was that when we install 150 packages, we then only fail one time instead of 150 times

Reply:
Done